config/schemas/config.yaml

```
@@ -7067,6 +7067,58 @@ properties:
         $ref: '#/$defs/kubernetesTopologySpreadConstraints'
       affinity:
         $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+  tektonPipelines:
+    title: Tekton Pipelines Config
+    description: |-
+      Configure Tekton Pipelines
```

Are one line string accepted?
```suggestion
    description: Configure Tekton Pipelines
```